public/app/features/dimensions/editors/FileUploader.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { FileDropzone, useTheme2, Icon, Button } from '@grafana/ui';
+import { getBackendSrv, config } from '@grafana/runtime';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+interface Props {
+  setNewValue: Dispatch<SetStateAction<string>>;
+}
+
+export function FileDropzoneCustomChildren({
+  primaryText = 'Upload file',
+  secondaryText = 'Drag and drop here or browse',
+}) {
+  const theme = useTheme2();
+  const styles = getStyles(theme);
+
+  return (
+    <div className={styles.iconWrapper}>
+      <Icon name="upload" size="xxl" />
+      <h3>{primaryText}</h3>
+      <small className={styles.small}>{secondaryText}</small>
+      <h6>Or</h6>
+      <Button icon="upload">Upload</Button>
+    </div>
+  );
+}
+export const FileUploader = (props: Props) => {
```

Comment:
Small nit

```suggestion
export const FileUploader = ({ setNewValue }: Props) => {
```

and remove line 28